ds3-autogen-java/src/main/java/com/spectralogic/ds3autogen/java/helpers/JavaHelper.java

```diff
@@ -341,11 +348,11 @@ public static String getResponseCodes(final ImmutableList<Ds3ResponseCode> respo
 
     public static String getModelVariable(final Element element) {
         final StringBuilder builder = new StringBuilder();
-        builder.append(indent(1) + "@JsonProperty(\"" + capFirst(element.getName()) + "\")\n");
+        builder.append(indent(1)).append("@JsonProperty(\"").append(capFirst(element.getName())).append("\")\n");
```

Comment:
Technically no, but since a `StringBuilder` is being used it's better to keep things consistent and use append.